main.cpp

```diff
@@ -1010,13 +730,371 @@ Part 1e - Step 19: Request a review
+    5) Material (std::string)
+3 things it can do:
+    1) Play individual notes
+    2) Be tuned to different tunings
+    3) Strum notes
+Thing 5) Wings
+5 properties:
+    1) Wing shape (std::string)
+    2) Wing length (int)
+    3) Ratio of wings length to width (int)
+    4) Angle of wing tilt (float)
+    5) Flap angle (float)
+3 things it can do:
+    1) Generate lift
+    2) Provide stability
+    3) Adjust to various conditions and scenarios
+
+Thing 6) Engines
+5 properties:
+    1) Number of engines (int)
+    2) Power output (float)
+    3) Fuel efficiency (float)
+    4) Emissions (float)
+    5) Size (int)
+3 things it can do:
+    1) Provide thrust
+    2) Generate elecricity
+    3) Control speed and altitude
+
+Thing 7) Cargo space
+5 properties:
+    1) Volume (float)
+    2) Weight (int)
+    3) Temperature (int)
+    4) Capacity (int)
+    5) Loading time (float)
+3 things it can do:
+    1) Transport
+    2) Store
+    3) Secure
+
+Thing 8) Tail
+5 properties:
+    1) Tail length (int)
+    2) Tail width (int)
+    3) Elevator angle (float)
+    4) Stabilizer angle (float)
+    5) Yaw control width (int)
+3 things it can do:
+    1) Create stability
+    2) Create lift
+    3) Control side to side movement
+
+Thing 9) Fuselage
+5 properties:
+    1) Length of fuselage (int)
+    2) Width of fuselage (int)
+    3) Height of fuselage (float)
+    4) Capacity of fuselage (int)
+    5) Drag from fuselage (float)
+3 things it can do:
+    1) Enclose passenger cabin and cargo
+    2) Support wings and tail
+    3) Maintain internal pressure
+
+Thing 10) Airplane
+5 properties: 
+    1) Wings
+    2) Engines
+    3) Cargo space
+    4) Tail
+    5) Fuselage
+3 things it can do:
+    1) Fly
+    2) Transport passengers and cargo
+    3) Communicate with air traffic control
+*/
+
+struct Cellphone
+{
+// 5 properties: 
+    // 1) Number of camera lens (int)
+    int numCameraLens = 2;
+    //     2) Manufacturer (std::string)
+    std::string manufacaturer = "Apple";
+    //     3) Ram (int)
+    int ram = 16;
+    //     4) Screen size (float)
+    float screenSize = 6.2f;
+    //     5) Amount of storage (int)
+    int storageAmount = 32;
+    
+    // 3 things it can do:
+    //     1) Make phone calls
+    void callNumber(std::string phoneNumber);
+    //     2) Take photos and videos
+    void takePhoto(); 
+    //     3) Conenct to the internet
+    void connectOnline(std::string wifiConnection);
+};
 
+struct MovieTheater
+{
+    // 5 properties: 
+    //     1) Number of theaters (int)
+    int numTheaters = 13;
+    //     2) Cost of small popcorn (double)
+    double costSmallPopcorn = 8.9;
+    //     3) Cost of one ticket (double)
+    double costOneTicket = 19.9;
+    //     4) Number of IMAX theaters (int)
+    int numImaxTheaters = 2;
+    //     5) Number of movies available (int)
+    int numMoviesAvailable = 15;
+    // 3 things it can do:
+    //     1) Sell movie tickets
+    void chargeForTicket(); 
```

Comment:
I'm marking this because:
1. your plain-english description is not an action, but is an ability or describes a state OR
2. the plain-english description is fine but the function name doesn't match OR
3. The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise